src/main/java/org/nazar/service/EmailService.java

```diff
@@ -0,0 +1,42 @@
+package org.nazar.service;
+
+import org.nazar.service.smtp.SmtpAuthenticator;
+
+import javax.mail.*;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+import java.util.Properties;
+
+public class EmailService {
+    private final Authenticator authenticator = new SmtpAuthenticator();
+
+    public static final String TO = "nazarvlk793@gmail.com";
+    public static final String FROM = "nazar.valko09@gmail.com";
+
+    public static final String PASSWORD = "gxib gftv jryu noyn";
+   public void sendEmail(String text) {
```

Comment:
In real project, you should add unit test for this method. But inside method, you use consts FROM and TO. What if you want to test another mail addresses? I think you should FROM, TO , subject and other params as args to the method.